gsl_programmation/tests/test_services.py

```diff
@@ -41,3 +44,114 @@ def test_update_montant(projet):
 
     assert simulation_projet.montant == new_montant
     assert simulation_projet.taux == 50.0
+
+
+### ProjetService
+
+
+@pytest.fixture
+def simulation() -> Simulation:
+    return SimulationFactory()
+
+
+@pytest.fixture
+def projets_with_assiette(simulation) -> list[Projet]:
+    for amount in [10_000, 20_000, 30_000]:
```

Comment:
À part ça j'aime bien éviter les listes quand je peux utiliser des tuples : pas besoin de mutabilité quand on n'a pas l'intention de faire muter des trucs.

```suggestion
    for amount in (10_000, 20_000, 30_000):
```

Reply:
C'est bien vrai !